sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -723,11 +727,20 @@ struct _pi_queue : _pi_object {
     // Level Zero command queue handles.
     std::vector<ze_command_queue_handle_t> ZeQueues;
 
+    // Immediate commandlist handles, one per Level Zero command queue handle.
```

Comment:
I am wondering if a std::map<ze_command_queue_handle_t, ze_command_list_handle_t> is a cleaner rep here. Minor nitpick.

Reply:
This representation is used so that all interfaces using commandlists remain the same in the rest of the plug, whether we are using standard or immediate commandlists. For example, getAvailableCommandList, executeCommandList.